evm/package.json

```diff
@@ -4,15 +4,44 @@
   "license": "MIT",
   "scripts": {
     "build": "truffle build",
-    "test": "truffle test",
-    "lint": "echo \"No linter configured for evm project\" &&  exit 0"
+    "depcheck": "echo 'chainlink' && depcheck --ignore-dirs=build/contracts || true",
+    "lint": "echo \"No linter configured for evm project\" &&  exit 0",
+    "solhint": "solhint ./contracts/**/*.sol",
+    "test": "truffle test"
   },
   "dependencies": {
+    "chainlink": "0.6.1",
+    "@ensdomains/ens": "^0.1.1",
+    "link_token": "^1.0.3",
+    "openzeppelin-solidity": "^1.12.0",
+    "cbor": "^4.1.1",
+    "eth-ens-namehash": "^2.0.8",
+    "ethereumjs-abi": "^0.6.7",
+    "ethereumjs-tx": "^1.3.7",
+    "ethereumjs-util": "^6.1.0",
+    "ethereumjs-wallet": "^0.6.2",
+    "ethjs": "^0.4.0",
+    "solc": "0.4.25",
+    "solidity-cborutils": "^1.0.4",
+    "truffle-contract": "^4.0.27",
     "web3": "^1.2.0"
   },
   "devDependencies": {
-    "@types/web3": "^1.0.0",
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
+    "@babel/plugin-proposal-export-namespace-from": "^7.2.0",
+    "@babel/plugin-proposal-throw-expressions": "^7.2.0",
```

Comment:
We can remove this from here and .babelrc, typescript doesnt support this syntax